app/components/embed/pdf_component.html.erb

```diff
@@ -2,9 +2,17 @@
   <%= render Embed::HeaderComponent.new viewer: viewer %>
   <% # removing 5 pixels from the body height to account for being embedded %>
   <div class='sul-embed-body sul-embed-pdf'>
-    <object data="<%= viewer.pdf_files.first %>" type="application/pdf" style="height: 100vh; width: 100%">
-      <p>Your browser does not support viewing PDFs.  Please <a href="<%= viewer.pdf_files.first %>">download the file</a> to view it.</p>
-    </object>
+    <% if viewer.available? %>
+      <object data="<%= viewer.pdf_files.first %>" type="application/pdf" style="height: 100vh; width: 100%">
+        <div style="padding: 10%;">
+          Your browser does not support viewing PDFs.  Please <a target="_blank" href="<%= viewer.pdf_files.first %>">download the file</a> to view it.
+        </div>
+      </object>
+    <% else %>
+       <div style="padding: 10%;">
+          <%= viewer.restriction_message %>
+       </div>
+    <% end %>
   </div>
```

Comment:
Show restriction messaging in place of PDF file (if there are restrictions that prevent viewing of content).  Stanford only restrictions will be dealt with in a separate PR since users will be able to login and to do something about it, while these restrictions prevent viewing in all cases.